src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="GET"
+      class="form"
+      data-qa="big"
+    >
+      <label for="big" class="search search-big">
```

Comment:
use one of the inputs as the basic case, and override necessary styles for other cases. You can add big input style to the basic input class and rewrite style for small input